themes/bootstrap3/templates/RecordTab/collectionlist.phtml

```diff
@@ -42,6 +42,16 @@
     </div>
   </div>
 
+  <?php if (!empty($params->getFilterList(false))): ?>
```

Comment:
I had to resolve a merge conflict here, and I'm not sure if I put this in the right place. Please double-check. It seems to function correctly in this position, but there seems to be a line of excessive whitespace above the controls:

![image](https://user-images.githubusercontent.com/309069/52231574-55b37b80-2888-11e9-91f9-3745f332898c.png)

Reply:
Looks like it's in right place. I tweaked the padding a bit, but is it enough?

Reply:
Hmm, I'm still seeing what looks like a full blank line between the search controls and the filter list. It might help to reduce the bottom margin on the search controls, but I'm not sure if that would mess up other displays....

Reply:
I've now tweaked the margins a bit.

Reply:
Looks much better to me -- thanks!